libs/expo/betterangels/src/lib/screens/Client/Profile/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Colors } from '@monorepo/expo/shared/static';
+import { useState } from 'react';
+import { MainScrollContainer } from '../../../ui-components';
+import { ClientProfileQuery } from '../__generated__/Client.generated';
+import PersonalInfo from './PersonalInfo';
+
+export default function Profile({
+  client,
+}: {
+  client: ClientProfileQuery | undefined;
```

Comment:
Can you please explain what's happening here? Is `client` the `ClientProfileQuery`? Or is it the response of the `ClientProfileQuery`? I'm just not familiar with this syntax 🙏

Reply:
either we should pass data.clientProfile and type es ClientProfileType or if we want to pass full response we give ClientProfileQuery Type.

Reply:
so, when it's used `PersonalInfo`, this is using the ClientProfileQuery Type?
```
  const fullName = `${client?.clientProfile.user.firstName ?? ''} ${
    client?.clientProfile.user.lastName ?? ''
  }`.trim();
```